python/run/miniAODToNTuple_cfg.py

```diff
@@ -150,6 +152,7 @@
 else:
     process.makingNTuples = cms.Path(
         # process.metFilters *
+        process.smearedElectrons *
```

Comment:
Are you smearing electrons also for real data?
I do not see where you remove the sequence from the path for real data.